mock/py/mockbuild/package_manager.py

```diff
@@ -221,6 +221,9 @@ def execute(self, *args, **kwargs):
         if not self.buildroot.mounts.essential_mounted:
             self.buildroot.mounts.mountall_essential()
             pm_umount = True
+
+        self.buildroot.mounts.mount_bootstrap()
```

Comment:
Should this be called only when bootstrap is enabled? The same for umount. The check can be as well in the method. But if I read this correctly, I do not see this check.

Reply:
Ahh, the check is in mock.py on 792. When bootstrap is not enabled, the mount are not added there. 
Ignore me.